adlfs/spec.py

```diff
@@ -1473,6 +1484,7 @@ def __init__(
         self.end = None
         self.start = None
         self.closed = False
+        self.loop = self.fs.loop or get_loop()
```

Comment:
Is this used anywhere?

Reply:
self.loop is expected by sync_wrapper.

Reply:
Thanks for the feedback on the context manager for the container client @TomAugspurger.  Agree that having this as a policy will help with clarity around the managing the asynchronous clients.